### PR TITLE
WEBITOOLS-40: Makes in-place available with add/deliver version

### PR DESCRIPTION
**Pull request by leon2Meudon92**
Allows the user to set the in-place option when application created created without using the  create application command

### CODE REVIEW

File: aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/AddVersionCommand.java [hunk 1]
```diff
@@ -150,7 +154,7 @@ public Integer call() {
         try {
             if (StringUtils.isBlank(applicationGuid)) {
                 log.info("Searching for application '{}' on AIP Console", applicationName);
-                applicationGuid = applicationService.getOrCreateApplicationFromName(applicationName, autoCreate, nodeName, domainName);
+                applicationGuid = applicationService.getOrCreateApplicationFromName(applicationName, autoCreate, nodeName, domainName, inPlaceMode);
```
Comment (skn33) [inline aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/AddVersionCommand.java:157]: you need to check if the application is really in inplace mode, if the application is already created

**Comment by skn33** [comment]
@leon2Meudon92  tests failed

File: aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/AddVersionCommand.java (hunk 1 above)
Comment (leon2Meudon92) [reply aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/AddVersionCommand.java:157]: Check done earlier here
inplaceMode = app == null ? isInPlaceMode() : app.isInPlaceMode();

**Comment by leon2Meudon92** [comment]
@skn decide not to add this by now, so close

**Pull request closed without merging**